Wikipedia/Code/NotificationsCenterAction.swift

```diff
@@ -18,4 +18,19 @@ struct NotificationsCenterActionData: Equatable {
     let url: URL?
     let iconType: NotificationsCenterIconType?
     let destinationText: String?
+    let actionType: RemoteNotificationActionType?
 }
+
+public enum RemoteNotificationActionType: String {
```

Comment:
I think it might be a bit cleaner if this was nested within `NotificationsCenterActionData` and called `LoggingLabel` or something like that, to make it clear that this is only used for analytics and doesn't power other logic in the app.